.env.schema

```
@@ -0,0 +1,25 @@
+# on build only the for build required variables get validated (good for docker build)
+# on prod only the runtime required variables get validated (good for docker run)
+# @type=enum(dev, ci, build, prod)
+APP_MODE=dev
+
+# True to use the Feedback control
+# @type=boolean
+USE_FEEDBACK=not(eq($APP_MODE, ci))
+
+# Gets calulcated from the APP_MODE
+# @type=boolean
+IS_RUNTIME=not(eq($APP_MODE, build))
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)